python/guide/module/gluon/init.md

```diff
@@ -161,3 +170,127 @@ net.add(nn.Dense(10, in_units=256))
+
+```{.python .input  n=9}
+# force_reinit ensures that the variables are initialized again, regardless of whether they were
+# already initialized previously.
+net.initialize(init=init.Normal(sigma=0.01), force_reinit=True)
+net[0].weight.data()[0]
+```
+
+If we wanted to initialize all parameters to 1, we could do this simply by
+changing the initializer to `Constant(1)`.
+
+```{.python .input  n=10}
+net.initialize(init=init.Constant(1), force_reinit=True)
+net[0].weight.data()[0]
+```
+
+If we want to initialize only a specific parameter in a different manner, we
+can simply set the initializer only for the appropriate subblock (or parameter)
+for that matter. For instance, below we initialize the second layer to a
+constant value of 42 and we use the `Xavier` initializer for the weights of the
+first layer.
+
+```{.python .input  n=11}
+net[1].initialize(init=init.Constant(42), force_reinit=True)
+net[0].weight.initialize(init=init.Xavier(), force_reinit=True)
+print(net[1].weight.data()[0,0])
+print(net[0].weight.data()[0])
+```
+
+### Custom Initialization
+
+Sometimes, the initialization methods we need are not provided in the `init`
+module. At this point, we can implement a subclass of the `Initializer` class
+so that we can use it like any other initialization method. Usually, we only
+need to implement the `_init_weight` function and modify the incoming NDArray
+according to the initial result. In the example below, we  pick a decidedly
+bizarre and nontrivial distribution, just to prove the point. We draw the
+coefficients from the following distribution:
+
+$$
+\begin{aligned}
+    w \sim \begin{cases}
+        U[5, 10] & \text{ with probability } \frac{1}{4} \\
+            0    & \text{ with probability } \frac{1}{2} \\
+        U[-10, -5] & \text{ with probability } \frac{1}{4}
+    \end{cases}
+\end{aligned}
+$$
+
+```{.python .input  n=12}
+class MyInit(init.Initializer):
+    def _init_weight(self, name, data):
+        print('Init', name, data.shape)
+        data[:] = nd.random.uniform(low=-10, high=10, shape=data.shape)
+        data *= data.abs() >= 5
+
+net.initialize(MyInit(), force_reinit=True)
+net[0].weight.data()[0]
+```
+
+If even this functionality is insufficient, we can set parameters directly.
+Since `data()` returns an NDArray we can access it just like any other matrix.
+A note for advanced users - if you want to adjust parameters within an
+`autograd` scope you need to use `set_data` to avoid confusing the automatic
+differentiation mechanics.
+
+```{.python .input  n=13}
+net[0].weight.data()[:] += 1
+net[0].weight.data()[0,0] = 42
+net[0].weight.data()[0]
+```
+
+## Tied Parameters
+
+In some cases, we want to share model parameters across multiple layers. For
+instance when we want to find good word embeddings we may decide to use the
+same parameters both for encoding and decoding of words. We discussed one such
+case when we introduced [Blocks](model-construction.md). Let's see how to do
```

Comment:
Link needs to be adjusted

Reply:
I've removed the link as there isn't a Blocks link. As mentioned in the other PR we should revisit to figure out how to link between pages. I'll add a SIM.

Reply:
I've removed the link as there isn't a Blocks link. As mentioned in the other PR we should revisit to figure out how to link between pages.